README.md

```diff
@@ -21,7 +21,7 @@ There are four packages you can install:
 - @jpmorganchase/uitk-icons contains SVG-based icons you can use in your
   application.
 
-Depending on the package manager you use, edit and run one of the following commands to install the packages you need:
+Depending on the `package manager` you use, edit and run one of the following commands to install the packages you need:
```

Comment:
is this necessary?

Reply:
Don't see why we should emphasise that?

Reply:
Apologies Zarko, this was for a demo to the wider content team.